plugins/package.json

```diff
@@ -0,0 +1,12 @@
+{
+    "name": "plugins",
+    "version": "0.0.0",
+    "license": "MIT",
+    "private": true,
+    "dependencies": {
+        "posthog-plugin-server": "^0.0.6"
```

Comment:
Wonder if we should hard-pin this version to avoid incompatibility when we change posthog-plugin-server later on

Reply:
We do use the `.lock`-file when installing the packages in this folder, but it can't hurt to be even stricter. Done.